app/models/chef.rb

```diff
@@ -1,4 +1,9 @@
 class Chef < ApplicationRecord
-   validates_presence_of :name
-   has_many :dishes
+  validates_presence_of :name
+  has_many :dishes
+  has_many :ingredients, through: :dishes
+
+  def unique_ingredients
+    ingredients.distinct
```

Comment:
Excellent encapsulation of this unique list. Adding the additional association definitely makes this logic easier